runtime/src/lib.rs

```diff
@@ -461,6 +482,30 @@ impl_runtime_apis! {
 				nonce,
 			).map_err(|err| err.into())
 		}
+
+		fn current_transaction_statuses() -> Option<Vec<TransactionStatus>> {
+			Ethereum::current_transaction_statuses()
+		}
+
+		fn current_block() -> Option<pallet_ethereum::Block> {
+			Ethereum::current_block()
+		}
+
+		fn current_receipts() -> Option<Vec<pallet_ethereum::Receipt>> {
+			Ethereum::current_receipts()
+		}
+
+		fn current_all() -> (
+			Option<pallet_ethereum::Block>,
+			Option<Vec<pallet_ethereum::Receipt>>,
+			Option<Vec<TransactionStatus>>
+		) {
+			(
+				Ethereum::current_block(),
+				Ethereum::current_receipts(),
+				Ethereum::current_transaction_statuses()
+			)
+		}
```

Comment:
Why was this stuff not present before? Is it because our hotfixes branch removes this stuff from the runtime API?

Reply:
Yes, we use the storage api in our fork and all those calls are for the runtime api.